npm/src/wrapper.ts

```diff
@@ -97,9 +99,9 @@ function getInt32Memory0(wasm) {
     return cachegetInt32Memory0;
 }
 
-let cachedTextDecoder = new TextDecoder('utf-8', { ignoreBOM: true, fatal: true });
+const lTextDecoder = typeof TextDecoder === 'undefined' ? module.require('util').TextDecoder : TextDecoder;
 
-cachedTextDecoder.decode();
```

Comment:
is it OK that you don't call `decode` anymore?

Reply:
Well, seems like this line was added by accident